may. 12, 2023 2:40:38 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:47)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:110)

may. 12, 2023 3:02:14 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:47)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:110)

may. 12, 2023 3:40:29 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.update
org.sqlite.SQLiteException: [SQLITE_ERROR] SQL error or missing database (near "WHERE": syntax error)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.throwex(DB.java:1112)
	at org.sqlite.core.NativeDB.prepare_utf8(Native Method)
	at org.sqlite.core.NativeDB.prepare(NativeDB.java:117)
	at org.sqlite.core.DB.prepare(DB.java:267)
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:44)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:30)
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:25)
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:34)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:155)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:135)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.update(BancoPreguntasDao.java:69)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:123)

may. 12, 2023 3:49:20 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_BUSY] The database file is locked (database is locked)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.throwex(DB.java:1112)
	at org.sqlite.core.NativeDB.prepare_utf8(Native Method)
	at org.sqlite.core.NativeDB.prepare(NativeDB.java:117)
	at org.sqlite.core.DB.prepare(DB.java:267)
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:44)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:30)
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:25)
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:34)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:155)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:135)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:150)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:42)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:110)

may. 12, 2023 3:59:53 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.update
org.sqlite.SQLiteException: [SQLITE_ERROR] SQL error or missing database (near "WHERE": syntax error)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.throwex(DB.java:1112)
	at org.sqlite.core.NativeDB.prepare_utf8(Native Method)
	at org.sqlite.core.NativeDB.prepare(NativeDB.java:117)
	at org.sqlite.core.DB.prepare(DB.java:267)
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:44)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:30)
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:25)
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:34)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:155)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:135)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.update(BancoPreguntasDao.java:69)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:123)

may. 12, 2023 4:16:55 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:47)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:111)

may. 12, 2023 4:20:11 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:47)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:111)

may. 12, 2023 4:21:40 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:47)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:111)

may. 12, 2023 4:22:31 A. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.BancoPreguntasDao.create
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.create(BancoPreguntasDao.java:47)
	at pe.edu.upeu.app.dao.BancoPreguntasDao.main(BancoPreguntasDao.java:111)

